package/features/environment.py

```diff
@@ -49,7 +49,7 @@ def _setup_context_with_venv(context, venv_dir):
     context.pip = str(bin_dir / "pip")
     context.python = str(bin_dir / "python")
     context.kedro = str(bin_dir / "kedro")
-    context.requirements_path = Path("requirements.txt").resolve()
+    context.requirements_path = Path("test_requirements.txt").resolve()
```

Comment:
This is because kedro extras requirements are in the test_requirements and not in requirements. So e2e_tests keep failing